src/Command.ts

```diff
@@ -27,6 +27,10 @@ const executors = {
     },
     clear: (invocation: Invocation, args: Array<string>): void => {
         setTimeout(() => invocation.terminal.clearInvocations(), 0);
+    },
+    exit: (invocation: Invocation, args: Array<string>): void => {
+        var app = require('remote').require('app');
+        app.quit();
```

Comment:
What is the best way to access `Application` to remove a terminal? Is there a direct way to access the application from within `Command`, or after getting an `Invocation`? Or, would it be better to create an ipc message to handle it in `ApplicationComponent` similar to [this](https://github.com/black-screen/black-screen/blob/075d22e68ca1a4017c8b71b963a049df65423e0e/src/views/ApplicationComponent.ts#L25)?